test/test_mimic_explainer.py

```diff
@@ -392,6 +394,29 @@ def test_explain_model_imbalanced_classes(self, mimic_explainer):
         assert np.isclose(surrogate_predictions, model_predictions).all()
         assert global_explanation.method == LIGHTGBM_METHOD
 
+    def test_explain_raw_feats_regression(self, mimic_explainer):
+        # verify that no errors get thrown when calling get_raw_feat_importances
```

Comment:
Is `get_raw_feat_importances` called under the hood here?

Reply:
yes, exactly, get_raw_feat_importances is called from here